modyn/models/gpt2/gpt2.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any
+
+import torch
+from torch import nn
+from transformers import AutoTokenizer, GPT2Config, GPT2LMHeadModel
+
+from modyn.models.coreset_methods_support import CoresetSupportingModule
+
+
+class Gpt2:
```

Comment:
Generally, we should use `GPT2`, not `Gpt2` for consistency. Also, can we move the addition of GPT2 into a separate PR?